presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -485,6 +492,10 @@ Drop a partition from the ``page_views`` table::
     WHERE ds = DATE '2016-08-09'
       AND country = 'US'
 
+Add an empty partition to the ``page_views`` table::
+
+    CALL system.create_empty_partition('web', 'page_views', ARRAY['ds', 'country'], ARRAY['2016-08-09', 'US']);
```

Comment:
Let's remove the semicolon since this is a single statement, and we don't have one for any of the other examples

Reply:
We should update the argument names in the procedure to be snake case. Since argument names are identifiers, declaring them as uppercase would require the user to quote them:
```sql
CALL system.create_empty_partition(schema => '...', table => '...', "partitionColumnNames" => '...', ...)
```

Reply:
How about calling them `partition_columns` and `partition_values`?

Reply:
This is what it would look like:
```sql
CALL system.create_empty_partition(
    schema => 'web',
    table => 'clicks', 
    partition_columns => array['ds'],
    partition_values => array['2018-05-03']);
```
I thought about shortening them, but `values` is a reserved word.